yt/go/yt/interface.go

```diff
@@ -1569,6 +1569,13 @@ type QueryTrackerOptions struct {
 	Stage *string `http:"stage,omitnil"`
 }
 
+type Secret struct {
```

Comment:
Let's add the field names in the "yson" tag for each field. Otherwise, the field names will be used as keys.